maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FeatureToggleEntity.java

```diff
@@ -0,0 +1,32 @@
+package gov.uk.courtdata.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name = "FEATURE_TOGGLE", schema = "TOGDATA")
+public class FeatureToggleEntity {
+
+  @Id
+  @Column(name = "ID", nullable = false)
```

Comment:
Looking at the FEATURE_TOGGLE table, ID uses a sequence. Should we keep the entity inline with the DB table please?

Reply:
Done. I'm not sure if the sequence name here needs to match one that already exists or not - I couldn't see anything in the db. If you could confirm that'd be great, thanks.

Reply:
Looking at the DB, the sequence we are using is ISEQ$$_316195